content/applications/productivity/iot/config/troubleshooting.rst

```diff
@@ -418,3 +418,68 @@ The cash drawer should be connected to the printer and the :guilabel:`Cash drawe
 be ticked in the :abbr:`PoS (Point of Sale)` configuration. To do so, navigate to
 :menuselection:`POS app --> 3-Dot Menu on the POS --> IoT Box section --> Edit --> Receipt Printer
 --> Cashdrawer checkbox`.
+
+Scale
+=====
+
+Scales play a crucial role in the checkout process, especially for products sold by weight, rather
+than fixed pricing.
+
+Set up Ariva S scales
+---------------------
+
+Odoo has determined that a specific setting in Ariva S series scales (manufactured by
+Mettler-Toledo, LLC.) needs modification, and a dedicated Mettler :abbr:`USB (Universal Serial
+Bus)`-to-proprietary RJ45 cable is required for the scale to function with Odoo's :abbr:`IoT
+(Internet of Things)` box.
+
+To correctly configure the scale for recognition by the :abbr:`IoT (Internet of Things)` box, follow
+this setup process for the Ariva S series scales.
+
+.. important::
+   It is crucial to use the official Mettler :abbr:`USB (Universal Serial Bus)`-to-RJ45 cable during
+   this process.
+
+Cable
+~~~~~
+
+The Mettler part number is 72256236 - :abbr:`USB (Universal Serial Bus)`-to-:abbr:`POS (Point of
+Sale)` cable. Contact Mettler, or a partner, to purchase an authentic cable. Note that **no other**
+cable outside of this Mettler cable works for this configuration. Using a serial-only cable attached
+to a serial-to-:abbr:`USB (Universal Serial Bus)` adapter is **not** effective.
+
+.. image:: troubleshooting/cable-mettler.png
+   :align: center
+   :alt: Authentic Mettler USB to POS cable, part number 72256236.
+
+Setup
+~~~~~
+
+Refer to Mettler's Setup Guide for Ariva S series scales during the following configuration: `Ariva
+Checkout Scale User's Guide <https://www.mt.com/dam/RET_DOCS/Ariv.pdf>`_.
+
+To begin, go to page 17 in the above manual for 'Setup'. This guide lists potential settings for the
+Ariva S series scales.
```

Comment:
This is a nitpick, since you are referencing the section name in the PDF, I'd prefer italics over single quote formatting. WDYT?
```suggestion
To begin, go to page 17 in the above manual for *Setup*. This guide lists potential settings for the
Ariva S series scales.
```